Backend/dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12
```

Comment:
Would consider changing the image used for the backend image. The default python docker image i quite large at over 1 GB. Maybe consider using the ```python:3.12-alpine``` image. 

I tested changing to it on my local machine and the swagger documentation as-well as the ping api call still worked. (python-alpine image is 90 MB so it does save considerable space and speed up first time building).

Reply:
Thanks, I have updated and tested using the alpine version instead